From: Rick Jones <rick.jones2@hp.com>
Date: Fri, 7 Oct 2011 19:13:28 -0400
Subject: [PATCH 12/49] net: Remove unnecessary driver assignments of
 ethtool_ringparam fields to zero
Patch-mainline: v3.2-rc1
Git-commit: 8b0c11679fd37522d8d34a76101319a085d80912 (partial)
References: bsc#909566 FATE#314306

Per comments from Ben Hutchings on a previous patch, sweep the floors
a little removing unnecessary assignments of zero to fields of struct
ethtool_ringparam in driver code supporting ethtool -g.

Signed-off-by: Rick Jones <rick.jones2@hp.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Gary Ching-Pang Lin <glin@suse.com>
---
 drivers/net/ethernet/marvell/sky2.c |    4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/net/ethernet/marvell/sky2.c
+++ b/drivers/net/ethernet/marvell/sky2.c
@@ -4110,13 +4110,9 @@ static void sky2_get_ringparam(struct ne
 	struct sky2_port *sky2 = netdev_priv(dev);
 
 	ering->rx_max_pending = RX_MAX_PENDING;
-	ering->rx_mini_max_pending = 0;
-	ering->rx_jumbo_max_pending = 0;
 	ering->tx_max_pending = TX_MAX_PENDING;
 
 	ering->rx_pending = sky2->rx_pending;
-	ering->rx_mini_pending = 0;
-	ering->rx_jumbo_pending = 0;
 	ering->tx_pending = sky2->tx_pending;
 }
 
